app/models/user.rb

```diff
@@ -1,8 +1,13 @@
 class User < ActiveRecord::Base
+  has_many :user_events
+  has_many :events, through: :user_events
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-         
-  def is_admin?
-    self.admin
+
+  validates_presence_of :first_name, :last_name, :email,
+                        :password, :password_confirmation
+
+  def admin?
```

Comment:
this method is predefined by rails itself http://makandracards.com/makandra/607-automagical-methods-for-boolean-attributes-in-activerecord (so you can delete it)